src/Carbon/CarbonInterval.php

```diff
@@ -522,6 +522,8 @@ public static function createFromFormat(string $format, ?string $interval)
             $interval = implode($match[1], $interval);
         }
 
+        $interval = $interval ?? '';
```

Comment:
Fixes `Tests\CarbonInterval\CreateFromFormatTest::testNulls()`.

Reply:
`testNulls()` does not trigger error on GitHub Actions for PHP 8.1, I guess they don't use the very last build:
https://github.com/briannesbitt/Carbon/runs/2665500828